test/scripts/generate-handlers-json.driver.ts

```diff
@@ -0,0 +1,50 @@
+import path from "path";
+import getEventsHandlersService, {
+  ComponentsMap,
+  EventHandler
+} from "../../scripts/generate-components-handlers/generate-handlers-json";
+
+const FIXTURE_PATH = path.join(__dirname, "./fixtures/$w-events.d.ts");
+
+export interface Driver {
+  given: {
+    mockDeclarationsFile: () => void;
+  };
+  get: {
+    events: () => ComponentsMap;
+    eventHandlersByComponentName: (name: string) => EventHandler[] | undefined;
+    componentHandlersNames: (componentName: string) => string[] | undefined;
+  };
+}
+
+const getDriver = (): Driver => {
+  let events: ComponentsMap;
+
+  return {
+    given: {
+      mockDeclarationsFile: () => {
```

Comment:
it makes sense to accept the fixture as arg and let the test control what is being mocked

Reply:
noice